chart/values.yaml

```diff
@@ -332,6 +332,8 @@ components:
       timeoutSeconds: 300
     insecureNoDomain: false
     chargebeeProviderOptionsFile: "/chargebee/providerOptions"
+    prebuildLimiter:
+      '*': 10
```

Comment:
Not very important, but this low value looks a little bit risky for an example config. 100 could be safer. 💭